chapters/01.md

```diff
@@ -4,7 +4,7 @@ Moments before the collision, Wint was dreaming of the North. Hanging heavily fr
 
-He'd been set up with the younger woman by an old friend from his days at The Post, who would've undoubtedly by now received a full recounting of his shortcomings of the night before. Having cancelled all his afternoon meetings, Wint had left his office on time, but after a few impromptu pints of dutch courage on the way to the restaurant he was already late and sweaty; not the best of entrances. He wasn't sure what pictures Sarah, their shared acquaintance, had shown his date to persuade her to meet him, but odds were they were of an older vintage. He'd been careful not to be photographed unnecessarily the last few years as his waistline had expanded and his formerly black hair had taken on a sombre shade of grey. That said, catching his reflection in the window of the restaurant as he smoked the last of his cigarette, he thought he wasn't that bad looking — he was tallish, recently groomed, his face cleanly shaven apart of a rakish little moustache and a seven-o-clock shadow, and underneath his peacoat and recently acquired layer of pudge he had good bones and the build of a former Cambridge lacrosse team captain — granted, it had been a good few years since. But as he was shown to the table at the brightly lit minimalist sushi joint he had chosen, his date's disappointed look made it clear she did not agree with his generous, lager-enhanced self assessment, and that the evening would not end in great romance, as they rarely had in recent years.
+He'd been set up with the younger woman by an old friend from his days at The Post, who would've undoubtedly by now received a full recounting of his shortcomings of the night before. Having cancelled all his afternoon meetings, Wint had left his office on time, but after a few impromptu pints of dutch courage on the way to the restaurant he was already late and sweaty; not the best of entrances. He wasn't sure what pictures Sarah, their shared acquaintance, had shown his date to persuade her to meet him, but odds were they were of an older vintage. He'd been careful not to be photographed unnecessarily the last few years as his waistline had expanded and his formerly black hair had taken on a sombre shade of grey. That said, catching his reflection in the window of the restaurant as he smoked the last of his cigarette, he thought he wasn't that bad looking — he was tallish, recently groomed, his face cleanly shaven apart from a rakish little moustache and a seven-o-clock shadow, and underneath his peacoat and recently acquired layer of pudge he had good bones and the build of a former Cambridge lacrosse team captain — granted, it had been a good few years since. But as he was shown to the table at the brightly lit minimalist sushi joint he had chosen, his date's disappointed look made it clear she did not agree with his generous, lager-enhanced self assessment, and that the evening would not end in great romance, as they rarely had in recent ye
```

Comment:
I think something has gone wrong here, the paragraph ends in "had in recent ye".

Reply:
Hmm yeah. Will check what’s up.